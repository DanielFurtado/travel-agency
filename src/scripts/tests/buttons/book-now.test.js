import React from 'react';
import { shallow } from 'enzyme';
import BookNowButton from '../../../components/buttons/book-now-button';

describe('BookNowButton', () => {
  const bookNowButton = shallow(<BookNowButton />);
  
  it('renders properly', () => {
    expect(bookNowButton).toMatchSnapshot();
  });
});