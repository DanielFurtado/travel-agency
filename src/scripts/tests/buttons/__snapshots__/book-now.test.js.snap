// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookNowButton renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BookNowButton />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "book now",
      "className": "btn-book-now",
      "id": undefined,
    },
    "ref": null,
    "rendered": "book now",
    "type": "button",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "book now",
        "className": "btn-book-now",
        "id": undefined,
      },
      "ref": null,
      "rendered": "book now",
      "type": "button",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
